test/run_test

```diff
@@ -127,13 +136,15 @@ function create_container() {
   echo "Created container $(cat $cidfile)"
 }
 
-function create_volume() {
-  local volume_dir
+
+create_volume_dir ()
+{
   volume_dir=`mktemp -d --tmpdir pg-testdata.XXXXX`
   setfacl -m u:26:-wx "$volume_dir"
   echo "$volume_dir"
```

Comment:
What does this echo do? When `create_volume_dir` is called output is not captured...